cgc/coclustering.py

```diff
@@ -57,6 +57,17 @@ class Coclustering(object):
     :type row_clusters_init: numpy.ndarray or array_like, optional
     :param col_clusters_init: Initial column cluster assignment.
     :type col_clusters_init: numpy.ndarray or array_like, optional
+
+    :Example:
+
+    >>> import numpy as np
+    >>> Z = np.random.randint(100, size=(10, 8)).astype('float64')
```

Comment:
Zeros can be problematic for co-/tri-clustering, let's maybe use:
```suggestion
    >>> Z = np.random.randint(1, 100, size=(10, 8)).astype('float64')
```